### PR TITLE
bundle-builds

### PR DESCRIPTION
**Pull request by mbiarnes**

### CODE REVIEW

File: elliott/elliottlib/cli/verify_cvp_cli.py [hunk 1]
```diff
@@ -81,20 +84,36 @@ async def verify_cvp_cli(runtime: Runtime, all_images, nvrs, include_content_set
...
         runtime.logger.info(f"Getting CVP test results for {len(nvr_builds)} image builds...")
-        nvr_results = await inspector.latest_sanity_test_results(nvr_builds.keys())
+        # are there bundle builds in the nvrs?
+        for k, v in nvr_builds.items():
+            runtime.logger.info(f"NVRS : {k}")
+            sub_string_1 = 'bundle'
+            sub_string_2 = 'metadata'
+            if sub_string_1 in k or sub_string_2 in k:
+                runtime.logger.info(f"The NVR is a bundle_nvr: {k}")
+                bundle_nvrs = nvr_builds
```
[the remark this answers is not in the document]
Comment (thegreyd) [reply elliott/elliottlib/cli/verify_cvp_cli.py:97]: We should initialize an empty bundle_nvrs list, so `bundle_nvrs = []` before line 91

File: elliott/elliottlib/cvp.py [hunk 2]
```diff
@@ -55,6 +55,21 @@ async def latest_sanity_test_results(self, nvrs: Iterable[str]) -> Dict[str, Opt
             nvr_results[nvr] = None  # missing result
         return nvr_results
 
+    async def latest_bundle_test_results(self, bundle_nvrs: Iterable[str]) -> Dict[str, Optional[Dict]]:
+        """ Get latest CVP test results for specified bundle build NVRs
```
Comment (thegreyd) [inline elliott/elliottlib/cvp.py:59]: In this function, we want to call this url for the nvrs that are passed in http://external-ci-coldstorage.datahub.redhat.com/cvp/cvp-redhat-operator-bundle-image-validation-test/ 
then parse the results and return

**Comment by mbiarnes** [comment]
@thegreyd @vfreex About expand elliott verify-cvp to operator bundles. This is working locally
using: ose-cluster-kube-descheduler-operator-container-v4.14.0-202401231734.p0.gc3ddfd6.assembly.stream OR
ose-cluster-kube-descheduler-operator-**metadata**-container-v4.14.0.202311211133.p0.gc3ddfd6.assembly.stream-3
All comments are welcome.

**Comment by mbiarnes** [comment]
can be closed since this pr https://github.com/openshift-eng/art-tools/pull/295/files does the same

**Pull request closed without merging**